### DISCUSSION TITLE
[Feature] add a link to the github somewhere on the website

### DISCUSSION THREAD

**Discussion by benjamin051000** [immich-app/immich#4287] [Feature Request]
### The feature

Hi! Immich is awesome. I periodically like to check the repository for progress/new versions/just to see what's going on. It would be convenient to have a clickable link somewhere on the site to take a user to the repo. Here are some spots I think may be acceptable locations: 
![Screenshot 2023-09-30 at 12-22-27 Photos](https://github.com/immich-app/immich/assets/34549440/1381dc98-2b1d-4cce-948e-29e398ca3bf2)

- Make the server version a clickable <a> tag that sends you either to that particular release, or to the overall repository site
- Perhaps an entry someplace in the user popup
- Additionally, a link someplace on the administration tab would be sufficient

In theory, this would be relatively trivial to implement. What do you all think? 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by danieldietzler** [comment]
Done! #4288

**Reply by benjamin051000** [reply]
That was fast! Thanks for adding that in. :D

**Reply by danieldietzler** [reply]
It was really a minor thing lol
